app/controllers/concerns/api_pagination.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ApiPagination
+  extend ActiveSupport::Concern
+  include Pagy::Backend
```

Comment:
Are there any places where we can reuse this? Maybe not in this PR, but if you could add a `todo` in them to use this concern it would make it easier to pick up for someone else later.

Reply:
It can be used in other places but as you say for another PR. I don't really want to add a TODO as it doesn't seem important or pressing enough and the functionality is not degraded or missing

Reply:
If we get this merged I'll pick this up in my next PR, touching those controllers anyway